packages/components/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 4.2.0 (Unreleased)
+
+### Deprecation
+
+- `wp.components.PanelColor` has been deprecated in favor of `wp.editor.PanelColorSettings`.
```

Comment:
In the context of `packages`, we should avoid references to WordPress-specific globals when possible (at least for changelog). Console messaging is trickier since we can't yet customize for the WordPress context, but it's a hope there we could.

See: #9753